src/Service.Tests/dab-config.MySql.json

```diff
@@ -23,2149 +23,2102 @@
-        "type": {
-          "singular": "Bookmarks",
-          "plural": "Bookmarks"
-        }
-      },
-      "rest": {
-        "enabled": true,
-        "path": null,
-        "methods": []
-      },
-      "permissions": [
+    {
+      "role": "authenticated",
+      "actions": [
         {
-          "role": "anonymous",
-          "actions": [
-            {
-              "action": "*",
-              "policy": {}
-            }
-          ]
-        },
+          "action": "*",
+          "policy": {}
+        }
+      ]
+    }
+  ],
+  "mappings": {
+    "__column1": "column1",
+    "__column2": "column2"
+  }
+},
+    "Bookmarks": {
+  "source": {
+    "object": "bookmarks",
+    "type": "table"
+  },
+  "graphql": {
+    "enabled": true,
+    "operation": null,
+    "type": {
+      "singular": "Bookmarks",
+      "plural": "Bookmarks"
+    }
+  },
+  "rest": {
+    "enabled": true,
+    "methods": []
+  },
+  "permissions": [
+    {
+      "role": "anonymous",
+      "actions": [
         {
-          "role": "authenticated",
-          "actions": [
-            {
-              "action": "*",
-              "policy": {}
-            }
-          ]
+          "action": "*",
+          "policy": {}
         }
       ]
     },
-    "MappedBookmarks": {
-      "source": {
-        "object": "mappedbookmarks",
-        "type": "table"
-      },
-      "graphql": {
-        "enabled": true,
-        "operation": null,
-        "type": {
-          "singular": "MappedBookmarks",
-          "plural": "MappedBookmarks"
+    {
+      "role": "authenticated",
+      "actions": [
+        {
+          "action": "*",
+          "policy": {}
         }
-      },
-      "rest": {
-        "enabled": true,
-        "path": null,
-        "methods": []
-      },
-      "permissions": [
+      ]
+    }
+  ]
+},
+    "MappedBookmarks": {
+  "source": {
+    "object": "mappedbookmarks",
+    "type": "table"
+  },
+  "graphql": {
+    "enabled": true,
+    "operation": null,
+    "type": {
+      "singular": "MappedBookmarks",
+      "plural": "MappedBookmarks"
+    }
+  },
+  "rest": {
+    "enabled": true,
+    "methods": []
+  },
+  "permissions": [
+    {
+      "role": "anonymous",
+      "actions": [
         {
-          "role": "anonymous",
-          "actions": [
-            {
-              "action": "*",
-              "policy": {}
-            }
-          ]
-        },
+          "action": "*",
+          "policy": {}
+        }
+      ]
+    },
+    {
+      "role": "authenticated",
+      "actions": [
         {
-          "role": "authenticated",
-          "actions": [
-            {
-              "action": "*",
-              "policy": {}
-            }
-          ]
+          "action": "*",
+          "policy": {}
         }
-      ],
-      "mappings": {
-        "id": "bkid",
-        "bkname": "name"
-      }
+      ]
     }
+  ],
+  "mappings": {
+    "id": "bkid",
+    "bkname": "name"
   }
 }
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
Given this file is auto-generated, manually adding new lines won't be a good idea.